fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/TypeRdfContext.java

```diff
@@ -58,71 +61,36 @@ public TypeRdfContext(final FedoraResource resource,
 
         //include rdf:type for primaryType, mixins, and their supertypes
         concatRdfTypes();
+        if (resource() instanceof NonRdfSource) {
+            // gather versionability info from the parent
+            if (resource().getNode().getParent().isNodeType("mix:versionable")) {
```

Comment:
It is indeed the parent's status that produces the child's triple. This is because they will always be the same (to my understanding). Do you think an inline comment is appropriate here?